clang/include/clang/Basic/Attr.td

```diff
@@ -2189,6 +2189,14 @@ def BTFTypeTag : TypeAttr {
   let LangOpts = [COnly];
 }
 
+def BPFFastCall : InheritableAttr,
+                  TargetSpecificAttr<TargetBPF> {
+  let Spellings = [Clang<"bpf_fastcall">];
+  let Subjects = SubjectList<[FunctionLike]>;
+  let Documentation = [BPFFastCallDocs];
+  let SimpleHandler = 1;
+}
```

Comment:
If we want to enforce attribute only for C lang, we can add `let LangOpts = [COnly];` in the above.

Reply:
Makes sense

Reply:
done